### PR TITLE
overhauls radiation

### PR DESCRIPTION
**Pull request by silicons**
:cl: 
refactor: radiation now uses a wave system
add: radiation can now contaminate items
add: geigers can now detect radiation contamination, and work from range
add: showers can decontaminate radioactive items
tweak: whole-level irradiation has been reworked; it'll probably end horribly. you have been warned.
balance: radiation damage and armor heavily reworked
/:cl:

### CODE REVIEW

File: code/__DEFINES/traits/_helpers.dm [hunk 1]
```diff
@@ -0,0 +1,69 @@
+
+// trait accessor defines
+#define ADD_TRAIT(target, trait, source) \
```
Comment (TheLordME) [inline code/__DEFINES/traits/_helpers.dm:3]: the \ is to keep it as a macro?

**Review by TheLordME — DISMISSED** [review]
Lets be real, its a silicons PR, if they leave the server dies, and if they make a mistake they fix it themselves

File: code/__DEFINES/traits/_helpers.dm (hunk 1 above)
Comment (Zandario) [reply code/__DEFINES/traits/_helpers.dm:3]: It's the newline indicator? It's **required**.

**Review by Zandario — APPROVED** [review]
I agree to the terms of service

**Pull request merged**